src/Features/Blockcore.Features.NodeHost/UI/Shared/MainLayout.razor

```diff
@@ -1,19 +1,23 @@
 ﻿@inherits LayoutComponentBase
 
     <Modal />
-    <div class="sidebar">
+
+    <div class="sidebar bg-dark">
         <NavMenu />
     </div>
 
-    <div class="main container border-left border-secondary">
-        <div class="container pt-3">
-            @Body
-        </div>
+    <div class="main">
+
         @* <div class="top-row p-4 border-bottom">
             <span class="float-right">
             <a href="/docs/index.html" target="_blank">Open API</a>
-            <a href="/ws-ui/index.html" target="_blank>Web Socket</a>
+            <a href="/ws-ui/index.html" target="_blank">Web Socket</a>
```

Comment:
Do we plan to show this? maybe as a modal or a new page (on a separate PR)

Reply:
I plan to revisit the layout at some point to add these links back in

Reply:
I plan to revisit the layout at some point to add these links back in